docs/develop/host-specific-api-model.md

```diff
@@ -0,0 +1,211 @@
+
+```js
+Excel.run(function (context) {
+    var sheetName = 'Sheet1';
+    var rangeAddress = 'A1:B2';
+    var myRange = context.workbook.worksheets.getItem(sheetName).getRange(rangeAddress);
+
+    myRange.load('address');
+
+    return context.sync()
+      .then(function () {
+        console.log (myRange.address);   // ok
+        //console.log (myRange.values);  // not ok as it was not loaded
+        });
+    }).then(function () {
+        console.log('done');
+}).catch(function (error) {
+    console.log('Error: ' + error);
+    if (error instanceof OfficeExtension.Error) {
+        console.log('Debug info: ' + JSON.stringify(error.debugInfo));
+    }
+});
+```
+
+> [!NOTE]
+> If you are only calling methods or setting properties on a proxy object, you don't need to call the `load()` method. The `load()` method is only required when you want to read properties on a proxy object.
+
+Just like requests to set properties or invoke methods on proxy objects, requests to load properties on proxy objects get added to the queue of pending commands on the request context, which will run the next time you call the `sync()` method. You can queue up as many `load()` calls on the request context as necessary.
+
+#### Scalar and navigation properties
+
+There are two categories of properties: **scalar** and **navigational**. Scalar properties are assignable types such as strings, integers, and JSON structs. Navigation properties are readonly objects and collections of objects that have their fields assigned, instead of directly assigning the property. For example, `name` and `position` members on the [Excel.Worksheet](/javascript/api/excel/excel.worksheet) object are scalar properties, whereas `protection` and `tables` are navigation properties.
+
+Your add-in can use navigational properties as a path to load specific scalar properties. The following code queues up a `load` command for the name of the font used by an `Excel.Range` object, without loading any other information.
+
+```js
+someRange.load("format/font/name")
+```
+
+You can also set the scalar properties of a navigation property by traversing the path. For example, you could set the font size for an `Excel.Range` by using `someRange.format.font.size = 10;`. You don't need to load the property before you set it.
+
+#### Calling `load` without parameters
+
+If you call the `load()` method on an object (or collection) without specifying any parameters, all scalar properties of the object (or all scalar properties of all objects in the collection) will be loaded. To reduce the amount of data transfer between the Excel host application and the add-in, you should avoid calling the `load()` method without explicitly specifying which properties to load.
+
+> [!IMPORTANT]
```

Comment:
I dunno this seems Excel specific?

Reply:
Agreed. I'm going to leave the section of exceptions there because I don't have another place to put it yet.

Reply:
Are we aware of any similar exceptions for non-Excel?

Reply:
@ElizabethSamuel-MSFT, Excel is the only one with a different `load` behavior because of the sheer number of properties in `Range` (and some customers load large Ranges through Graph).